scalding-core/src/main/scala/com/twitter/scalding/bdd/TBddDsl.scala

```diff
@@ -68,7 +68,7 @@ trait TBddDsl extends FieldConversions with TypedPipeOperationsConversions {
 
       val outputPipe = operation(inputPipes) map { Tuple1(_) }
 
-      outputPipe.write(TypedTsv[Tuple1[OutputType]]("output"))
+      outputPipe.write(FixedPathTypedDelimited[Tuple1[OutputType]]("output", "\t"))
```

Comment:
Actually, I think the easiest fix is to define a simple mock `TypeDescriptor` since this code actually is not using any features of the macro. Then we shouldn't hit the macro path:
```scala
implicit val td: TypeDescriptor[Tuple1[OutputType]] = new TypeDescriptor[Tuple1[OutputType]] {
  def converter = TupleConverter.singleConverter
  def setter = TupleSetter.singleSetter // I think these names are close. ;)
  def fields = new Fields("item")
}
```
if this is in scope, the macro should not be hit and we'll be okay. Since this source is not written to disk, it doesn't matter.

PS: I'm not sure anyone uses this test framework, and I'm a bit unsure we should keep it around, but let's do something quick.